include/usearch/index.hpp

```diff
@@ -2721,8 +2731,13 @@ class index_gt {
         using slot_level_allocator_t = typename dynamic_allocator_traits_t::template rebind_alloc<slot_level_t>;
         buffer_gt<slot_level_t, slot_level_allocator_t> slots_and_levels(size());
 
+        // Progress status
+        std::atomic<bool> do_tasks{true};
```

Comment:
It's better to store the atomic error in an `std::atomic<char const *>`, like in other palces.

Reply:
This function doesn't return anything, so there is object to assign the error